### PR TITLE
Add chat_group.clear and update permissions of chat_message.create.

### PR DESCRIPTION
**Pull request by reiterl**
Add some tests,

### CODE REVIEW

File: openslides_backend/action/actions/chat_group/clear.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+@register_action("chat_group.clear")
+class ChatGroupClear(ChatEnabledMixin, UpdateAction):
+    """
+    Action to clear a chat group.
+    """
+
+    model = ChatGroup()
+    schema = DefaultSchema(ChatGroup()).get_default_schema(
+        title="Schema for clear a chat group.",
+        additional_required_fields={"chat_group_id": required_id_schema},
```
Comment (jsangmeister) [inline openslides_backend/action/actions/chat_group/clear.py:22]: I would think the `chat_group_` prefix is implicitly given by the action name, so only `id` as the parameter name should suffice. (@GabrielInTheWorld)

Comment (jsangmeister) [reply openslides_backend/action/actions/chat_group/clear.py:22]: (then you can just use the normal update schema here)

File: openslides_backend/action/actions/chat_group/clear.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    )
+    permission = Permissions.Chat.CAN_MANAGE
+    permission_id = "chat_group_id"
```
Comment (jsangmeister) [inline openslides_backend/action/actions/chat_group/clear.py:25]: ... and remove this line

File: openslides_backend/action/actions/chat_group/clear.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    def get_updated_instances(self, action_data: ActionData) -> ActionData:
+        for instance in action_data:
+            chat_group = self.datastore.get(
+                FullQualifiedId(self.model.collection, instance["chat_group_id"]),
+                ["chat_message_ids"],
+            )
+            delete_action_data = []
+            for id_ in chat_group.get("chat_message_ids", []):
+                delete_action_data.append({"id": id_})
+            self.execute_other_action(ChatMessageDelete, delete_action_data)
```
Comment (jsangmeister) [inline openslides_backend/action/actions/chat_group/clear.py:36]: execute after loop to potentially remove the amount of calls

**Review by jsangmeister — CHANGES_REQUESTED** [review]

File: openslides_backend/action/actions/chat_group/clear.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/action/actions/chat_group/clear.py]: Done.

File: openslides_backend/action/actions/chat_group/clear.py (hunk 2 above)
Comment (reiterl) [reply openslides_backend/action/actions/chat_group/clear.py]: Done.

File: openslides_backend/action/actions/chat_group/clear.py (hunk 3 above)
Comment (reiterl) [reply openslides_backend/action/actions/chat_group/clear.py]: Done.

**Review by jsangmeister — APPROVED** [review]

**Pull request merged**